Lib/importlib/_bootstrap_external.py

```diff
@@ -1646,6 +1646,15 @@ def _install(_bootstrap_module):
     "-ppc-linux-gnuspe.": "-powerpc-linux-gnuspe.",
     "-ppc64-linux-gnu.": "-powerpc64-linux-gnu.",
     "-ppc64le-linux-gnu.": "-powerpc64le-linux-gnu.",
+    # The above, but the other way around:
```

Comment:
Have you considered something like `_ARCH_MAP.update({v: k for k, v in _ARCH_MAP.items()})` instead of hardcoded reversed values? I mean, these names are very sensitive and it's very easy to do a typo here and an automatic update with the reversed keys/values might be less error-prone. Another point of view is that we probably never change this patch.

Reply:
In fact I have considered this, but decided not to do that for the following reasons:

 - I've considered this list to be fairly stable
 - hardcoding the values should be slightly "faster" (I realize that it makes no real difference, it is just a feel-more-right thing)
 - when a consumer debugs this patch, it makes more sense to me to see all the pairs at once

That said, your approach is likely as valid as mine. I don't really mind either way.

Reply:
Your reasons sound reasonable, keep the list as is.